npm/ng-packs/packages/oauth/src/lib/strategies/auth-password-flow-strategy.ts

```diff
@@ -40,13 +36,23 @@ export class AuthPasswordFlowStrategy extends AuthFlowStrategy {
   }
 
   async init() {
-    if (!getCookieValueByName(this.cookieKey) && localStorage.getItem(this.storageKey)) {
-      this.oAuthService.logOut();
-    }
+    this.checkRememberMeOption(this.localStorageService);
 
     return super.init().then(() => this.listenToTokenExpiration());
   }
 
+  private checkRememberMeOption(localStorageService: AbpLocalStorageService) {
+    const accessToken = this.oAuthService.getAccessToken();
+    const expireDate = this.oAuthService.getAccessTokenExpiration();
+    const currentDate = new Date().getTime();
+    const rememberMe = getRememberMe(localStorageService);
+
+    if (accessToken && expireDate < currentDate && rememberMe !== 'true') {
```

Comment:
For expiration date check, try to use any helper function form `oauth2-oidc` package. If it's not exits please add a function to service as `isTokenExpired`

Reply:
there is a method `hasValidAccessToken()`. but it doesn't work properly. even the accessToken expires it returns true.

Reply:
It shouldn't be like that it must works correct let's check together